web/styles/message_edit_history.css

```diff
@@ -1,37 +1,22 @@
-#message-edit-history {
-    .message_top_line {
-        float: right;
-    }
-
-    .date_row > span {
+.message-edit-history-container {
+    .header-body {
         display: flex;
         align-items: center;
-        white-space: nowrap;
+        flex-direction: row;
+        justify-content: space-between;
+        gap: 5px;
 
-        &::before,
-        &::after {
-            width: 100%;
-            margin: 0;
+        @media (width < $lg_min) {
+            display: block;
         }
     }
 
-    .message_time {
-        position: static;
-    }
-
-    .message_author {
-        position: relative;
-    }
-
-    .author_details {
-        display: block;
-        font-size: 12px;
-        padding: 1px;
-        text-align: right;
-    }
-
-    .messagebox-content {
-        padding: 0 10px;
+    .message-edit-history-list {
+        & h2 {
+            font-size: 1.1em;
```

Comment:
Can you explain where these values come from? Are there other headings in the UI that you're replicating here? I have the same question with the styles on `#message-history-error` below (which uses `rem` units instead of the `em` units here).

Reply:
The values for `.message-edit-history-list` came from `draft-list`. 
https://github.com/zulip/zulip/blob/ef5cdff9d98199355b70cac9a0657c2fe2da5c16/web/styles/drafts.css#L58-L63

previously the `message edit history` was shown using dialog based models which had font size of `1rem`, so I have taken the values from there:
https://github.com/zulip/zulip/blob/ef5cdff9d98199355b70cac9a0657c2fe2da5c16/web/styles/modal.css#L97-L10

Just to confirm, are you happy with these current values, or would you be open to some suggestions?

Reply:
It's probably not worth messing with them for now, but do leave a comment pointing to the styles that these are based on. That will help a future contributor who might be tasked with cleaning this up.

Reply:
sure!